src/plugins/intel_cpu/src/nodes/scaled_attn.cpp

```diff
@@ -77,7 +77,7 @@ struct MHAKernel {
     }
 
     PlainTensor<uint8_t> causal_mask;
-    bool select_nfltmax_at_0;  // set attn_score to -FLT_MAX when causal_mask[...] equal to this
+    bool select_nfltmax_at_0 = false;  // set attn_score to -FLT_MAX when causal_mask[...] equal to this
```

Comment:
Is Coverity reporting this line?
this class has `MHAKernel() = default;` all members if got default ctor will be initialized.